DependencyInjection/Configuration.php

```diff
@@ -85,6 +86,10 @@ private function getClassNode()
                     ->cannotBeEmpty()
                     ->defaultValue('Gedmo\\Blameable\\BlameableListener')
                 ->end()
+                ->scalarNode('ip_traceable')
+                    ->cannotBeEmpty()
+                    ->defaultValue('Gedmo\\IpTraceable\\IpTraceableListener')
```

Comment:
this setting should be removed as I don't want to introduce new class parameters anymore (it was a bad idea to introduce them)